src/common/styles/default-theme-palette.ts

```diff
@@ -26,4 +26,7 @@ export const DefaultThemePalette: IPartialTheme = {
         black: '#1d1d1d',
         white: '#ffffff',
     },
+    semanticColors: {
+        primaryButtonBackground: '#106ebe',
```

Comment:
I'm confused... should this color be '#0078d4' ??

Reply:
No, `#106ebe` is intentiona; per the latest figma for both the android app and the component figma for web, it's supposed to be communication-primary (like it was originally in web via the old `common.scss` stylee) in both places